.travis.yml

```
@@ -9,6 +9,7 @@ services:
 
 script:
   - docker build -t theconnman/docker-hub-rss .
+  - docker build -t theconnman/docker-hub-rss -f Dockerfile-armv7 .
```

This `docker build` step will default to building as the latest tag, which will overwrite the built file in the previous line unless it is smart enough to see the new OS architecture. Do you know if it is smart enough to know that?

Otherwise we need this.
```suggestion
  - docker build -t theconnman/docker-hub-rss:latest-armv7 -f Dockerfile-armv7 .
```